test/images/netproxy/netproxy.go

```diff
@@ -0,0 +1,235 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
yeah "netproxy" is just very generic.

On Mon, Sep 14, 2015 at 10:20 PM, Abhi Shah <notifications@github.com>
wrote:

> In test/images/netproxy/netproxy.go
> <https://github.com/kubernetes/kubernetes/pull/13697#discussion_r39475374>
> :
>
> > +Copyright 2014 The Kubernetes Authors All rights reserved.
> > +
> > +Licensed under the Apache License, Version 2.0 (the "License");
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +package main
>
> I called it netproxy, because we can use it to exec and also dial other
> endpoints through it.
> So it acts as a proxy to shell or other endpoints..
>
> are you concerned that is sounds similar to kubeproxy ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13697/files#r39475374>.
>